analyzer/src/main/kotlin/managers/PIP.kt

```diff
@@ -292,7 +294,8 @@ class PIP : PackageManager() {
         virtualEnvDir.safeDeleteRecursively()
 
         // PIP does not support lock files, so hard-code "allowDynamicVersions" to "true".
-        return ProjectAnalyzerResult(true, project, packages.map { it.toCuratedPackage() }.toSortedSet())
+        val config = AnalyzerConfiguration(false, true)
```

Comment:
Same as for Gradle; this also hard-codes `ignoreToolVersions` in the analyzer result.

Reply:
Thanks for catching this, done.